content/SCALE/SCALETutorials/Shares/MixedModeShares.md

```diff
@@ -0,0 +1,238 @@
+
+3. Scroll down to the **Permissions** widget. Click **Edit** to open the **Edit ACL** screen.
+
+4. Check the **Access Control List** to see if the AD group you created is on the list and has the correct permissions. If not, add this Access Control Entry (ACE) item.
+
+   a. Enter **Group** in the **Who** field or use the dropdown list to select **Group**.
+
+   b. Type or select the appropriate group in the **Group** field.
+
+   c. Verify **Full Control** displays in **Permissions**. If not, select it from the dropdown list.
+
+   d. Click **Save Access Control List** to add the ACE item or save changes.
+
+See [Permissions]({{< relref "PermissionsScale.md" >}}) for more information on editing dataset permissions.
+{{< /expand >}}
+
+### Creating the SMB Share
+
+To create the SMB share, go to **Shares**.
+
+1. Click on **Windows Shares (SMB)** to select it and then click **Add**. The **Add SMB** configuration screen displays the **Basic Options** settings.
+
+   {{< trueimage src="/images/SCALE/Shares/AddShareBasicOptions.png" alt="Add SMB Basic Options" id="Add SMB Basic Options" >}}
+
+2. Enter the SMB share **Path** and **Name**.
+
+   Enter the path or use the <span class="material-icons">arrow_right</span> icon to the left of **<span class="material-icons">folder</span>/mnt** to locate the dataset you created for the multiprotocol share.
+
+   The **Name** is the SMB share name, which forms part of the share pathname when SMB clients perform an SMB tree connect.
+   Because of how the SMB protocol uses the name, it must be less than or equal to 80 characters. It cannot have invalid characters as specified in Microsoft documentation MS-FSCC section 2.1.6.
+   If you do not enter a name, the share name becomes the last component of the path.
+   If you change the name, follow the naming conventions for:
+   * [Files and directories](https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions)
+   * [Share names](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-fscc/dc9978d7-6299-4c5a-a22d-a039cdc716ea)
+
+3. Select **Multi-protocol (NFSv4/SMB) shares** from the **Purpose** dropdown list to apply pre-determined **Advanced Options** settings for the share.
+
+4. (Optional) Enter a **Description** to help explain the share purpose.
+
+5. Select **Enabled** to allow sharing of this path when the SMB service is activated.
+   Leave it cleared if you want to disable the share without deleting the configuration.
+
+6. If needed, use **Advanced Options** to set up guest access, read only access, to set up allowed and denied hosts. or to optimize the SMB share for Apple OS.
+   See [Adding SMB Shares]({{< relref "AddSMBShares.md #configuring-share-advanced-options-settings" >}}) for more information.
+
+7. Click **Save** to create the share and add it to the **Shares > Windows (SMB) Shares** list.
+
+### Creating the NFS Share
```

Comment:
same comments here as in SMB share. If the current instructions cover this with the only change being the name of the dataset to use or a permission/setting option, then try to use a link to that procedure and only call out the difference.